src/rgw/rgw_tracer.h

```diff
@@ -0,0 +1,75 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab ft=cpp
+
+#ifndef TRACER_WRAPPER_H
+#define TRACER_WRAPPER_H
```

Comment:
we already have a wrapper for this stuff in `common/tracer.*` used by the osd, and i'd rather not duplicate that. if we need features that the existing wrappers don't support, can we extend those instead?

Reply:
its not a matter of features, the code there should not have been merged to the main branch (e.g. https://github.com/ceph/ceph/blob/master/src/common/tracer.cc#L28)

would recommend doing it right, to be "production ready", in the RGW, and then replacing the common/osd code

Reply:
> its not a matter of features, the code there should not have been merged to the main branch (e.g. https://github.com/ceph/ceph/blob/master/src/common/tracer.cc#L28)

what's the issue there? it falls back to a default config if it doesn't find one from the dev environment

> would recommend doing it right, to be "production ready", in the RGW, and then replacing the common/osd code

ok, has this been discussed with Deepika and the core team?